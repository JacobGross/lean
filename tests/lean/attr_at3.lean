namespace bla
definition f (a : nat) := a + 1
attribute f [reducible] at foo
print f
end bla


section
  open foo
  print bla.f
end

print bla.f

namespace foo
  print bla.f
end foo
