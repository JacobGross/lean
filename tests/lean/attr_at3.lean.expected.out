definition bla.f : ℕ → ℕ :=
λ (a : ℕ), a + 1
definition bla.f [reducible] : ℕ → ℕ :=
λ (a : ℕ), a + 1
definition bla.f : ℕ → ℕ :=
λ (a : ℕ), a + 1
definition bla.f [reducible] : ℕ → ℕ :=
λ (a : ℕ), a + 1
